lib/smart_answer_flows/marriage-abroad/outcomes/_services_and_fees.govspeak.erb

```diff
@@ -7,34 +7,34 @@
              collection: calculator.services,
              as: :service,
              locals: { calculator: calculator } %>
-<% end %>
 
-<% unless calculator.country_without_consular_facilities? ||
-   calculator.ceremony_country == 'indonesia' ||
-   calculator.ceremony_country == 'monaco' ||
-   calculator.ceremony_country == 'france' ||
-   calculator.ceremony_country == 'new-caledonia' ||
-   calculator.ceremony_country == 'wallis-and-futuna' ||
-   calculator.ceremony_country == 'french-guiana' ||
-   calculator.ceremony_country == 'french-polynesia' ||
-   calculator.ceremony_country == 'guadeloupe' ||
-   calculator.ceremony_country == 'martinique' ||
-   calculator.ceremony_country == 'mayotte' ||
-   calculator.ceremony_country == 'reunion' ||
-   calculator.ceremony_country == 'st-pierre-and-miquelon' %>
+  <% unless calculator.country_without_consular_facilities? ||
+     calculator.ceremony_country == 'indonesia' ||
+     calculator.ceremony_country == 'monaco' ||
+     calculator.ceremony_country == 'france' ||
+     calculator.ceremony_country == 'new-caledonia' ||
+     calculator.ceremony_country == 'wallis-and-futuna' ||
+     calculator.ceremony_country == 'french-guiana' ||
+     calculator.ceremony_country == 'french-polynesia' ||
+     calculator.ceremony_country == 'guadeloupe' ||
+     calculator.ceremony_country == 'martinique' ||
+     calculator.ceremony_country == 'mayotte' ||
+     calculator.ceremony_country == 'reunion' ||
+     calculator.ceremony_country == 'st-pierre-and-miquelon' %>
 
-  <% if calculator.ceremony_country == 'vietnam' %>
-    <%= render partial: 'link_to_consular_fees_pay_in_sterling_or_local_currency.govspeak.erb',
-               locals: { calculator: calculator } %>
+    <% if calculator.ceremony_country == 'vietnam' %>
+      <%= render partial: 'link_to_consular_fees_pay_in_sterling_or_local_currency.govspeak.erb',
+                 locals: { calculator: calculator } %>
 
-  <% elsif calculator.ceremony_country != 'cambodia' &&
-    !(calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?) %>
-    <%= render partial: 'link_to_consular_fees.govspeak.erb',
-               locals: { calculator: calculator } %>
+    <% elsif calculator.ceremony_country != 'cambodia' &&
+      !(calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?) %>
+      <%= render partial: 'link_to_consular_fees.govspeak.erb',
+                 locals: { calculator: calculator } %>
+    <% end %>
   <% end %>
-<% end %>
 
-<% if calculator.services.any? && calculator.services_payment_partial_name.present? %>
-  <% partial = 'payment_information_partials/' + calculator.services_payment_partial_name + '.govspeak.erb' %>
-  <%= render partial: partial, locals: { calculator: calculator } %>
+  <% if calculator.services_payment_partial_name.present? %>
```

Comment:
I don't really understand the commit note explanation for this change. I think part of the problem is there is a double-negative in the sentence. Also it's not clear to me whether or not this commit is changing the behaviour. The commit note makes it sound as if it *is*, but my guess is that it's not.